api/src/main/resources/workbench.yaml

```diff
@@ -71,6 +71,36 @@ paths:
           schema:
             $ref: "#/definitions/BugReport"
 
+  /api/v1/fail/fiveHundred:
```

Comment:
While I'm glad you tested with this, I don't think it should actually be a part of the API. Can we remove it now?

Reply:
I'm going to disagree on this one. I think these endpoints will be helpful to us while we develop and, for exactly the same reason, will be helpful to future developers wishing to develop against our API. It gives them a clear and simple way to cause our system to report an error so they can discover how we format them and, therefore, how they can parse them to do something useful with the information. I wish more APIs did this.

Reply:
Once the retry logic is working as desired, I will bet you $100 nobody ever uses these methods  again. :)

It ought to be possible to test the retry logic with a mock service on the client, right?

I personally wouldn't leave this code in the server API, but if you're going to leave it here, please remove the UI code that uses it... we don't need to pull down extra code just used for testing purposes to our end user's browsers.

Reply:
I think it's convenient and useful, but since it's not production code I don't think we should have it as part of the production server. I'd love to convert it to a convenient stub you can install in tests, for example, though.

Reply:
I don't think we're communicating adequately about this point. Let's discuss face-to-face.